src/Raven.Embedded/DatabaseOptions.cs

```diff
@@ -1,9 +1,13 @@
-﻿namespace Raven.Embedded
+﻿using Raven.Client.ServerWide;
+
+namespace Raven.Embedded
 {
     public class DatabaseOptions
     {
         public string DatabaseName { get; set; }
```

Comment:
Now when we have DatabaseRecord then I'm not sure if we need this. We should drop this or change DatabaseRecord to `Action<DatabaseRecord>` to modify the record which database name was set using the `DatabaseName` property.

Reply:
I like the action better, yes.